docs/guides/single-fetch.md

```diff
@@ -367,7 +351,7 @@ This poses an interesting conundrum for [resource routes][resource-routes] which
 
 If a resource route is intended for consumption by internal Remix APIs, we _want_ to be able to leverage the `turbo-stream` encoding to unlock the ability to stream down more complex structures such as `Date` and `Promise` instances. However, when accessed externally, we'd probably prefer to return the more easily consumable JSON structure. Thus, the behavior is slightly ambiguous if you return a raw object in v2 - should it be serialized via `turbo-stream` or `json()`?
 
-To ease backwards-compatibility and ease the adoption of the Single Fetch future flag, Remix v2 will handle this based on whether it's accessed from a Remix API or externally. In the future Remix will require you to return your own JSON response if you do not want raw objects to be streamed down for external consumption.
+To ease backwards-compatibility and ease the adoption of the Single Fetch future flag, Remix v2 will handle this based on whether it's accessed from a Remix API or externally. In the future Remix will require you to return your own [JSON response][returning-response] if you do not want raw objects to be streamed down for external consumption.
 
 The Remix v2 behavior with Single Fetch enabled is as follows:
```

Comment:
I can't comment where I want to, but down below I find these examples a little confusing. Since we're going to be deprecating `json` we should probably update these examples to use `new Response` instead, right?

I get that some instances we're trying to show a migration path, but I think it's a bit confusing to recommend using `json` when we also talk about how we're going to remove it in rr7